### ISSUE TITLE
Rename `time_explosion`

### ISSUE THREAD

**Issue by andrewfullard** [tardis-sn/tardis#2394]
`time_explosion` is not clearly defined and leads to confusion. It should be renamed to something more accurate e.g. "time_of_spectrum" (because it is the time at which the spectrum is generated).

**Comment by wkerzendorf** [comment]
I thought about this recently. I think it is not wrong in the sense that it relates to time in relation to the explosion itself (as that is needed to use homology). I feel `time_of_spectrum` is correct but that does not show what zero point it links to. I'm not sure if `time_explosion` is the right way forward but I'm not sure `time_of_spectrum` is either.

**Comment by chvogl** [comment]
I agree with @wkerzendorf that time_of_spectrum is somewhat ambiguous regarding the zero point.

**Comment by andrewfullard** [comment]
The example was definitely intended to provoke discussion!

**Comment by rudrakshnalbalwar** [comment]
Hi @andrewfullard , @wkerzendorf 
 I noticed the discussion about renaming time_explosion. I’d love to contribute to this issue. Based on the concerns raised, should we consider time_since_explosion as it explicitly conveys the reference point? Also, I can go through the codebase and update references accordingly. Let me know how I can help!!

**Comment by rudrakshnalbalwar** [comment]
> Hi [@andrewfullard](https://github.com/andrewfullard) , [@wkerzendorf](https://github.com/wkerzendorf) I noticed the discussion about renaming time_explosion. I’d love to contribute to this issue. Based on the concerns raised, should we consider time_since_explosion as it explicitly conveys the reference point? Also, I can go through the codebase and update references accordingly. Let me know how I can help!!

@wkerzendorf , @andrewfullard ??

**Comment by andrewfullard** [comment]
It's a good option and something we will discuss internally

**Comment by rohit2p** [comment]
can i work on this issue?? @andrewfullard

**Comment by andrewfullard** [comment]
> can i work on this issue?? [@andrewfullard](https://github.com/andrewfullard)

As I noted, this is something we are discussed internally
